infrastructure/router/router.go

```diff
@@ -0,0 +1,29 @@
+package router
+
+import (
+	"github.com/gin-gonic/gin"
+	"github.com/wmaldonadoc/academy-go-q42021/config"
+	"github.com/wmaldonadoc/academy-go-q42021/interface/controller"
+)
+
+func NewRouter(c controller.AppController) {
+	port := config.GetEnvVariable("PORT")
+
+	router := gin.Default()
+
+	mainGroup := router.Group("/api/v1")
+	{
+		pokemonRoutes := mainGroup.Group("/pokemons")
+		{
+			pokemonRoutes.GET("/:id", func(context *gin.Context) { c.Pokemon.GetById(context) })
+		}
+
+		healthRoutes := mainGroup.Group("/health")
+		{
+			healthRoutes.GET("/", func(context *gin.Context) { c.Health.GetServiceHealth(context) })
+		}
+	}
```

Comment:
Although this is not an error, Go's best practices suggest using opening brackets at the same level as the definition.

If wanted to use them this way, please be consistent among all the files.

```suggestion
	mainGroup := router.Group("/api/v1") {
		pokemonRoutes := mainGroup.Group("/pokemons") {
			pokemonRoutes.GET("/:id", func(context *gin.Context) { c.Pokemon.GetById(context) })
		}

		healthRoutes := mainGroup.Group("/health") {
			healthRoutes.GET("/", func(context *gin.Context) { c.Health.GetServiceHealth(context) })
		}
	}
```